docker-compose.yml

```diff
@@ -51,9 +57,76 @@ services:
       WAIT_HOSTS: redis:6379, postgres:5432, camunda:8080
       WAIT_TIMEOUT: 300
       WAIT_SLEEP_INTERVAL: 5
+      PLUGIN_REGISTRY_URL: "http://registry:8080"
+  backend:
+    image: ghcr.io/ust-quantil/qhana-backend:main
+    volumes:
+      - experiments:/app/data
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      # remove the following line if you're running the plugin-runner and worker outside of docker
+      QHANA_URL_MAPPING: '{"(?<=^|https?://)(localhost|host.docker.internal):5005": "qhana-plugin-runner:8080", "(?<=^|https?://)localhost": "host.docker.internal"}'
```

Comment:
We need a better solution for this mess in the future... (but I am not sure if there is a better solution)